src/views/Main.test.jsx

```diff
@@ -0,0 +1,68 @@
+import { ThemeProvider, createTheme } from '@mui/material/';
+import { CartProvider } from '../context/CartContext';
+import { screen, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import App from '../App';
+
+export const theme = createTheme({
```

Comment:
Really awesome job in implementing your theme along with these tests. The behavior you're testing is fluid and easy to follow.